src/scss/2022/settings/_font-size.scss

```
@@ -0,0 +1,10 @@
+$sizes: (
+  'heading-1': 4.5rem,
+  'heading-2': 3.5rem,
+  'heading-3': 2.5rem,
+  'heading-4': 2rem,
+  'section-description': 1.75rem,
+  'body-xlarge': 1.5rem,
+  'body-large': 1.25rem,
+  'body-default': 1.125rem
```

The Figma design calls for 17px, which translates to 1.0625rem
```suggestion
  'body-default': 1.0625rem
```

---

Well, you caught me trying to slide past my irrational dislike for odd numbers and decimal points larger than 3. 😉

---

Hey, I agree! I'm on your side here - don't shoot the messenger! 😉